nasa/openmct
# [No CLA Yet] Add link to the User's Guide in About Dialog (#4966)

Closes #4966 

### Describe your changes:
This pull request adds a link to the User's Guide from About Dialog.

To test:
1. Start the application
2. Click on the OpenMCT logo
3. Click on the link to the User's Guide

The expected result is opening a new browser tab with the guide in PDF format from the [OpenMCT site](https://nasa.github.io/openmct/documentation/).

### All Submissions:


### Author Checklist

* [x] Tests included and/or updated with changes?
* [x] Testing instructions included in associated issue OR is this a dependency/testcase change?

### Reviewer Checklist

* [ ] Reviewer has tested changes by following the provided instructions?
* [x] Appropriate automated tests included?
